tests/model_serving/model_runtime/vllm/basic_model_deployment/test_granite_3_1_2b_instruct.py

```
@@ -0,0 +1,75 @@
+import pytest
+import structlog
+import time
+
+from tests.model_serving.model_runtime.vllm.utils import (
```

Flake8 (7.3.0)

[error] 'tests.model_serving.model_runtime.vllm.utils.parse_vllm_logs' imported but unused

(F401)